p/themes/Origine/origine.css

```diff
@@ -697,53 +760,59 @@ a.btn {
 
 /*=== Index menu */
 .nav_menu {
+	padding: 5px 0;
 	background: #fafafa;
 	border-bottom: 1px solid #aaa;
 	text-align: center;
-	padding: 5px 0;
 }
 
 /*=== Feed articles */
 .flux {
-	border-left: 2px solid #aaa;
 	background: #fafafa;
+	border-left: 2px solid #aaa;
 }
+
 .flux:hover {
 	background: #fff;
 }
+
 .flux.current {
-	border-left: 2px solid #0062BE;
+	background: #fff;
+	border-left: 2px solid #0062be;
 }
+
 .flux.not_read {
-	border-left: 2px solid #FF5300;
-	background: #FFF3ED;
+	background: #fff3ed;
```

Comment:
@Alkarex Ah, I see the problem. I merged two `.flux.current` selectors but now this one overrides that `#fff`.

Wrt #2604.

I'll fix it, unless you want to.

Reply:
Go ahead 👍